www/toucan/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem         1.0
+PortGroup github   1.0
+
+github.setup       toucansites toucan 1.0.0-alpha.1
```

Comment:
standard ports should point to stable, released versions. That's not what this tag suggests, so perhaps wait until upstream has released an actual version?